website/docs/guides/bloblang/about.md

```diff
@@ -106,11 +106,11 @@ root.new_doc.type = $foo
 
 ### Metadata
 
-Benthos messages contain metadata that is separate from the main payload, in Bloblang you can modify the metadata of the resulting message with the `meta` assignment keyword. Metadata values of the resulting message are referenced within queries with `@`:
+Benthos messages contain metadata that is separate from the main payload, in Bloblang you can modify the metadata of the resulting message with the `meta` assignment keyword. Metadata values of the resulting message are referenced within queries with the `@` operator or the [`metadata()` function][blobl.functions.metadata]:
 
 ```coffee
 # Reference a metadata value
-root.new_doc.bar = @kafka_topic
+root.new_doc.bar = @kafka_topic # Or `@.kafka_topic` or `metadata("kafka_topic")`
```

Comment:
Thinking to maybe add a separate paragraph about `@.`, since it might not be immediately obvious to people that they can add a period after `@` to call methods and such (or extract fields which have spaces in them).